src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDataTable.java

```diff
@@ -250,17 +250,18 @@ else if (val instanceof OffsetTime)
     public synchronized Map<Integer, SQLServerDataColumn> getColumnMetadata() {
         return columnMetadata;
     }
-
+    
     public String getTvpName() {
         return tvpName;
     }
 
     /**
      * Retrieves the column meta data of this data table.
-     * 
+     * @deprecated
```

Comment:
Should give details like -> `@deprecated  As of release 6.1.x, replaced by {@link #[YOUR_METHOD]}` 
Also give reasons for deprecation which will useful for all contributors.